packages/database/src/modelClasses/Task.js

```diff
@@ -47,17 +48,21 @@ const formatValue = async (field, value, models) => {
       return assignee.full_name;
     }
     case 'repeat_schedule': {
-      if (!value || !value?.frequency) {
+      if (!value || value.freq === undefined || value.freq === null) {
         return "Doesn't repeat";
       }
 
-      return `${value.frequency.charAt(0).toUpperCase()}${value.frequency.slice(1)}`;
+      const frequency = Object.keys(RRULE_FREQUENCIES).find(
+        key => RRULE_FREQUENCIES[key] === value.freq,
+      );
+
+      if (!frequency) {
+        return "Doesn't repeat";
+      }
+
+      return `${frequency.charAt(0)}${frequency.slice(1).toLowerCase()}`;
```

Comment:
It's not very obvious what is going on here. Could you add a comment or example of the output format to help?